utils/ixdtf/src/parsers/records.rs

```diff
@@ -102,80 +102,76 @@ impl From<bool> for Sign {
 #[non_exhaustive]
 #[derive(Debug, Clone, Copy, PartialEq)]
 pub struct UTCOffsetRecord {
-    /// The `Sign` value of the `UtcOffsetRecord`
+    /// The `Sign` value of the `UtcOffsetRecord`.
     pub sign: Sign,
-    /// The hour value of the `UtcOffsetRecord`
+    /// The hour value of the `UtcOffsetRecord`.
     pub hour: u8,
     /// The minute value of the `UtcOffsetRecord`.
     pub minute: u8,
     /// The second value of the `UtcOffsetRecord`.
     pub second: u8,
-    /// Any nanosecond value of the `UTCOffsetRecord`
+    /// Any nanosecond value of the `UTCOffsetRecord`.
     pub nanosecond: u32,
 }
 
-/// The resulting record of a `Duration` parse.
+/// The resulting record of parsing `Duration` string.
 #[non_exhaustive]
```

Comment:
Thought: The set of allowable fields is part of the IXDTF specification, and we expect clients to read from this struct and its inner enums, which means they need to match on it. This is probably a case where we should make the struct be exhaustive.